Correct table header for PCE and non PCE scenarios (#56)
mulesoft/docs-general: modules/ROOT/pages/deployment-options.adoc

include::_attributes.adoc[]
endif::[]

Anypoint Platform provides multiple deployment options, each of which differs based on data hosting requirements, the location and management of Anypoint Platform, and the individual Anypoint Platform components included: 

* US cloud
+
The complete Anypoint Platform. All data is hosted within the US.
* EU cloud
+
Includes all US cloud components, but differs from the US cloud in that all data is hosted within the EU. Refer to xref:eu-control-plane::index.adoc[About the EU Control Plane] for additional information.
* MuleSoft Government Cloud
+

> FedRAMP moderate authorized version of US cloud. Refer to xref:gov-cloud::index.adoc[About Government Cloud] for additional information.

Reviewer: Will your audience know what "moderate authorized version" means?

* Anypoint Platform PCE
+
A single-tenant version of US cloud managed by your IT team. Anypoint Platform PCE supports on-premises and cloud installation and is specifically designed for customers with strict regulatory or compliance requirements that limit the use of cloud solutions. Refer to xref:private-cloud::index.adoc[About Anypoint Platform Private Cloud Edition] for additional information.  

== Anypoint Platform Location and Management
Your organization's needs determine where Anypoint Platform functionality is deployed and who manages it. Anypoint Platform functionality can be classified into two main categories:

* Control plane
+ 
Anypoint Platform control plane functionality includes the design, engagement, and management capabilities of Anypoint Platform, such as Anypoint Design Center, Anypoint Management Center, and Anypoint Exchange.
* Runtime plane
+

> Anypoint Platform runtime plane functionality includes the runtime engine (Mule) and runtime services, such as Anypoint Connectors, Anypoint Object Store, and messaging that allow you to scale the power of Mule and Anypoint Platform, and is where applications are deployed.

Reviewer: ```suggestion
Anypoint Platform runtime plane is where applications are deployed. Its functionality includes the runtime engine (Mule) and runtime services (such as Anypoint Connectors, Anypoint Object Store, and messaging) that enable you to scale the power of Mule and Anypoint Platform.
```
Adding the parens limits the awkwardness of verb agreement with "enable."

> Anypoint Platform control plane and runtime plane functionality can be deployed on-premises, in the cloud, or in a hybrid environment in the following configurations:

Reviewer: active voice?
```suggestion
You can deploy Anypoint Platform control plane and runtime plane functionality on-premises, in the cloud, or in a hybrid environment:
```
However, note that this intro list does not mimic the order in which the listed items are described, and it should.

* CloudHub
+
MuleSoft hosts and manages public cloud infrastructure for both control and runtime planes. 
* Hybrid
+

> MuleSoft hosts and manages public cloud infrastructure for the control plane, with the runtime plane deployed in a hybrid configuration: both Anypoint Runtime Fabric and on CloudHub.

Reviewer: ```suggestion
MuleSoft hosts and manages public cloud infrastructure for the control plane, with the runtime plane deployed in a hybrid configuration: both in Anypoint Runtime Fabric and on CloudHub.
```
